src/nxdoc/web-ui/web-ui-functional-testing.md

```diff
@@ -18,7 +18,7 @@ The advantages of this approach are threefold:
 - it makes it easier to write tests based on requirements, specifications, and acceptance criteria.
 - it facilitates test writing by people other than developers.
 
-Cucumber is a crucial element of our testing stack, promoting Behavior-Driven Development (or **BDD** for short). Cucumber tests are written in [Gherkin](https://docs.cucumber.io/gherkin/reference/), which are then translated into JavaScript instructions that run commands using the WebDriver protocol. Tests are written in **feature** files (ending with the `.feature` extension), which are composed by a set of **Scenarios** with one or more **Steps**. Steps common to more than one scenario are usually placed in a **Background**, which is run before every scenario. Steps must begin with one of the following keywords: *Given*, *When*, *Then*, *And*, *But*. Which keyword is used only affects it's meaning to readers, although the *Given* keyword is commonly used for provisioning steps.
+Cucumber is a crucial element of our testing stack, promoting Behavior-Driven Development (or **BDD** for short). Cucumber tests are written in [Gherkin](https://docs.cucumber.io/gherkin/reference/), which are then translated into JavaScript instructions that run commands using the WebDriver protocol. Tests are written in **feature** files (ending with the `.feature` extension), which is composed of a set of **Scenarios** with one or more **Steps**. Steps common to more than one scenario are usually placed in a **Background**, which is run before every scenario. Steps must begin with one of the following keywords: *Given*, *When*, *Then*, *And*, *But*. Which keyword is used only affects its meaning to readers, although the *Given* keyword is commonly used for provisioning steps.
```

Comment:
Since we are talking about `tests` written in files, so plural, we should use something like `which are composed of a set of **Scenarios**`